Core/Core/Dashboard/OfflineSync/ViewModel/DashboardOfflineSyncProgressCardViewModel.swift

```diff
@@ -35,7 +49,7 @@ class DashboardOfflineSyncProgressCardViewModel: ObservableObject {
 
     /**
      - parameters:
-        - offlineModeInteractor: This is used to determine if the feature flag is turned on. If it's off then
+     - offlineModeInteractor: This is used to determine if the feature flag is turned on. If it's off then
```

Comment:
This won't produce the expected documentation:
<table>
<tr><th>Before</th><th>After</th></tr>
<tr>
<td><img src="https://github.com/instructure/canvas-ios/assets/72396990/a3d7ff19-31ab-4920-8824-8d77d3f04a9b" maxHeight=500></td>
<td><img src="https://github.com/instructure/canvas-ios/assets/72396990/05b6335d-4c45-4b3e-8131-24eb1885fb16" maxHeight=500></td>
</tr>
</table>